menu_tools/utils/utils.py

```diff
@@ -63,7 +63,12 @@ def get_branches(ntuple_path: str, tree: str, obj: str):
         else:
             prefix = "L1PhaseII/"
 
-    obj_branches = [x.removeprefix(prefix + obj) for x in all_branches if obj in x]
+        ## nano
```

Comment:
I would leave the indentation level as before (i.e. not put this code in the `with` block) as there is no operation here anymore requiring the file to be opened.

Reply:
done in https://github.com/cms-l1-dpg/Phase2-L1MenuTools/pull/65